pkg/rescaffold/migrate.go

```diff
@@ -287,6 +287,8 @@ func getAPIResourceFlags(resource resource.Resource) []string {
 		args = append(args, "--resource")
 		if resource.API.Namespaced {
 			args = append(args, "--namespaced")
+		} else {
```

Comment:
@antonincms 

Can you please run `make lint-fix` to fix the linter issue in your PR so that we can get this one merged?

Reply:
I completely forgot, sorry, it's done !